src/utils/signing.ts

```diff
@@ -144,15 +140,16 @@ export async function signAgent(wallet: Wallet, action: any, isMainnet: boolean)
     );
 }
 
-async function signInner(wallet: Wallet | HDNodeWallet, data: any): Promise<Signature> {
-    const signature = await wallet.signTypedData(data.domain, data.types, data.message);
-    return splitSig(signature);
+async function signInner(data: any): Promise<Signature> {
+    // const signature = await wallet.signTypedData(data.domain, data.types, data.message);
+    // return splitSig(signature);
+    return { r: '0x', s: '0x', v: 0 };
```

Comment:
`signInner` returns hard coded or empty or invalid signature and has no real use can be removed or commented